<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_openCalendar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 172, 172);
border-radius: 15px;
border-color: black;
border-style: outset;
border-width: 2px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>расписание</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_allDoctors">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 172, 172);
border-radius: 15px;
border-color: black;
border-style: outset;
border-width: 2px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>все врачи</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_allPatients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 172, 172);
border-radius: 15px;
border-color: black;
border-style: outset;
border-width: 2px;
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>все пациенты</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
